lutris/gui/lutriswindow.py

```diff
@@ -73,6 +73,7 @@ def __init__(self, application, **kwargs):
         super().__init__(
             default_width=width,
             default_height=height,
+            window_position=Gtk.WindowPosition.NONE,
```

Comment:
Great ! You may want to merge just this line then, to be clear (i don't know what's the default is)